public/assets/js/picks.js

```diff
@@ -69,8 +69,8 @@ function dropdownClicked() {
             let idHomeString = "#gm" + i +"home";
             let idAwayString = "#gm" + i + "away";
 
```

Comment:
I hope you don't mind me doing some work on the .js file.  It was the only way I could find to style a couple of things for the Picks table.
I'm pretty sure what I did won't affect anything but let me know if I'm wrong about that.